tests/pkcs1v15.rs

```diff
@@ -0,0 +1,51 @@
+use pkcs8::DecodePrivateKey;
+use signature::Signer;
+
+use rsa::pkcs1v15::SigningKey;
+use rsa::RsaPrivateKey;
+
+const RSA_PRIV_PEM: &str = r#"-----BEGIN PRIVATE KEY-----
+MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQCxfecMGXkCgfz9
+ECLQlvTEYToty+m+RjFJufAp2ZvfL+jLNZKjqNvyRHbmvFZG24vFL0IkN6/e3k/H
+grTZoBJAdZ/sNtzNCiW7VHyTv4g6r9qi3rxmh7EDgqjN/HJb+EIMzskQqBipjrPV
+YiwBW8xRhwSfR4Drc7Oo++4kcDNi5he/+V6dyBszeDZaBSFEWCJaySgV5KUaOKIr
+tm1S+Gq0PMnf4ffixmyvM7COGxYY54Ij8j9ardhDYzO5MTsYV+5xkW8mbKRKFDgB
+VUzH6Pubp/k2j35fSA1ESw3WSLEEUaQz0oya5oWP3OqthMXTASWHdCyXOijTN/5K
+M7AVfR+BAgMBAAECggEAAfVEnkKNrjRm6RfLeoefzabQ8H5mZr35fzm4541KBe1O
+YSeHJZ0QSBzvPt5nCUMUyKkbzrB6lTfYo3fSrO/yI5nomL+GMhDhl6537V1YkGUu
+QUu5wlvO3R4M5LDbRh54BBIn77IbC4vSehEqMR2lJGUb7PHp/y74lW2qE+bU+ktQ
+ziTSSkk6vHh0xi1J+LXsl/OQVzokyLlK3hw9gpmKjv5MdbzmfaBfN0calZQEIbUY
+5jwv7bQV8yifbGl6elSnrM+aZ1y70RcvpafDW3Cy0XEXgetvE6iVlHQq9i5SWQDn
+daiRTeZB5+x4gv7sem73t7v2IVcoxw+DRoX29IV9DQKBgQC+ikg8HhlVEPkKeY6M
+2gxy2DFfi9t1NS5S4nkEtXuafIFEFtGvnEfxqIFDPjf/EqazBMWxyecXOy3QB5I/
+m5xUVeixl6s4QbGp0yIJ4Up9WW1iCJwke16qR5sM+HBPHyyn8QJrJLvahFQWSUL8
+0R4qWsMtrQz/M5EdHvM2CST51wKBgQDueAmASr2jL+I6K4GA8OIP3S6EoTvRDFDq
+la7ZRPMFI/ReeDSr8IBOiT1Gec9dOu4X8dI0YEMosObueyCdXpbGjqL/yGwGeVXm
+wbTZqsKElR2S7Sy2oGcAL2kKTaJgAnGoJQ+GNYZxnrLPn5aYudm4torNlzgY521m
+SlNte7j2ZwKBgD1pjnyp6tpLHSmuIi7tvKNrYQZ9ql8to9+dFQpagBs70+IhRLZI
+NbwC5p7It/2jgk0i1aFHBQ+syEhLmGyd+BYKlnRfARWhvuSQo2Dx1zNtll7JjaII
+jWtupt/YV0J5NshuWqwz68QzcHK6bDfWItXd6RiYtR6v/S6YoiUhb2SDAoGAWMT+
+ZG9uIG0WvaisUJ9ax6UCSATtC2iWRnzp1Z9BB8zWjsQYd1jTqmiUEsQNWRsyvl8a
+bLzYwzD3Bhd3kJ1BazB1pNmqvM42F+xLsitKtaf8llELsAAN15fOdjNoGm1OedUx
+7jfGkE0PKxKxMLEeRxBMozODgmHD66wHWl4QAEkCgYBWmmPKfUbCj+LUW+fKp8fx
+PhyvmJOm9dVkmyGOr8c28DVJZFzK1WCWn6r/sc5gfbir8McWvoy5ztFT8nUQAGmw
+ZKxu535xdbIQdh+XwB03HqcrsJYPVSzHz2lV8X3DVZ9v0ckQrA3Akq6mlWZbs72A
+TAVCoLQUA76jqB4eLsvCNQ==
+-----END PRIVATE KEY-----"#;
+
+// simple but prevent regression - see https://github.com/RustCrypto/RSA/issues/329
+#[cfg(feature = "pem")]
+#[test]
+fn signature_stringify() {
+    let private_key = RsaPrivateKey::from_pkcs8_pem(RSA_PRIV_PEM).unwrap();
+    let signing_key = SigningKey::<sha2::Sha256>::new(private_key);
+
+    let bytes: &[u8] = &[160, 185, 22, 183, 47, 48, 187, 224, 87, 122, 90, 214];
```

Comment:
Perhaps use a NUMS value, like `b"example"`

Reply:
We should construct a byte array that happen to with leading zero. I'll check if `b"example"` works.

Reply:
@tarcieri Updated b081defc9ea792b221c14af546faa7a24838f737